concepticondata/concepticon.tsv

```diff
@@ -3000,3 +3000,25 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3004	FORMER	Time	Coming before in time.	Property	
 3005	FAMILIAR	Emotions and values	Knowing somebody or something well.	Property	
 3006	SPACE (AVAILABLE)	Spatial relations	Available space to place things.	Person/Thing	
+3007	EUCALYPT	Agriculture and vegetation	Tree or wood for plants with capsule fruiting bodies belonging to closely related genera found across Australia.	
+3008	DEATH ADDER	Animals	Venomous snake of genus Acanthopis.	
+3009	ECHIDNA	Animals	Egg-laying mammal of the Tachyglossidae family.	
+3010	WHITE COCKATOO	Animals	Large white cockatoo, species C. galerita.	
+3011	WAGTAIL	Animals	Passerine bird of genus Motacilla.	
+3012	GRUB	Animals	Larva of Coleoptera order.	
+3013	BANDICOOT	Animals	Marsupial of Peramelemorphia order.	
+3014	BLACK COCKATOO	Animals	Black cockatoo, usually the Palm cockatoo.	
+3015	BLACK DUCK	Animals	Pacific black duck (Anas superciliosa).	
+3016	CROSS COUSIN	Kinship	Cousin from a parent's opposite-sex sibling (in relation to parallel cousin).	
+3017	PARALLEL COUSIN	Kinship	Cousin from a parent's same-sex sibling (in relation to cross cousin).	
+3018	FATHER'S FATHER	Kinship	Male grandparent of the father side (in relation to mother's father).	
```

Comment:
Did you make sure we don't have this already in the form of Grandfather (of male)? We should have it, as the Chinese lists also make this distinction, i.e., grandfather from father's side vs. grandfather from mother's side.

Reply:
I've just checked: indeed it is there, as PATERNAL GRANDFATHER, MATERNAL GRANDMOTHER and MATERNAL GRANDFATHER (apparently, PATERNAL GRANDMOTHER is missing). Must have been a problem with my query, will correct it.

Reply:
This is just what normally happens: We have list a) stating that it is "grandfather (paternal)", and then a new list b) stating that it is father's father. If you are not aware of list a), this can be accidentally be put into different concept sets, but now, that we spotted this, our algorithm for automatic linking will grow stronger and also find father's father if we compare against a new list.